cl/corpus_importer/tasks.py

```diff
@@ -2266,7 +2271,7 @@ def get_document_number_for_appellate(
 
     # Document numbers from documents with attachments have the format
     # 1-1, 1-2, 1-3 in those cases the document number is the left number.
-    document_number_split = document_number.split("-")
+    document_number_split = re.split(r"[-.]", document_number)
```

Comment:
I think the comment above this line could use an update. it took me a second to remember that ACMS uses (.) instead of  (-) for attachments.